graphql_compiler/tests/test_post_processing.py

```diff
@@ -340,3 +340,24 @@ def test_convert_complex(self):
 
         post_process_mssql_folds(query_output, output_metadata)
         self.assertEqual(query_output, expected_result)
+
+    def test_convert_invalid_result(self):
+        """Test invalid result throws error.
+
+                {
+                    Animal {
+                        in_Animal_ParentOf @fold{
+                            name @output(out_name: "child_names")
+                        }
+                    }
+                }
+                """
```

Comment:
This indentation seems a bit strange, assuming intentional?

Also perhaps a bit more context on the significance of the query might be useful.

Reply:
This was bad copy pasting! The query isn't really important to the test - I just thought it was a bit easier to understand what was going on if there was an example query that could have produced those results.

Reply:
Could we clarify in the docstring that this query is intended to be an example input that could have produced those results? I had a guess that was what was going on, but explicit is better than implicit.

Reply:
Also, if you don't mind, `@fold{` should be `@fold {` per the pretty-printer.